core/src/components/segment/segment.tsx

```diff
@@ -16,7 +16,9 @@ import { createColorClasses, hostContext } from '../../utils/theme';
     ios: 'segment.ios.scss',
     md: 'segment.md.scss'
   },
-  shadow: true
+  shadow: {
+    delegatesFocus: true
+  }
```

Comment:
If we add delegatesFocus do we still need the explicit "Enter" key listener below?

Reply:
Yes, if I remove the `Enter` listener then nothing happens on `Enter` keyup. 

I am guessing this is because all selection logic is handled in the parent `ion-segment` with an `ionSelect` event, rather than the child button's `onClick` or `onKeyUp`.